sw/airborne/mcu_periph/uart.h

```diff
@@ -36,8 +36,13 @@
 #define UART_RX_BUFFER_SIZE 128
 #endif
 #ifndef UART_TX_BUFFER_SIZE
+#ifdef STM32F4  //the F4 has enough memory, and the PX4 bootloader needs more then 128
```

Comment:
Which is why I took the compromise road here and changed it only for the f4 :)

So, what would you like me to do here? Move the define to the uart_arch.h?